src/Mvc/test/Mvc.FunctionalTests/HtmlGenerationTest.cs

```diff
@@ -96,6 +108,50 @@ public async Task EnumValues_SerializeCorrectly()
             Assert.Equal($"Vrijdag{Environment.NewLine}Month: FirstOne", response, ignoreLineEndingDifferences: true);
         }
 
+        [ConditionalTheory]
+        [MemberData(nameof(WebPagesDataNonHelix))]
+        [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/10423")]
+        public async Task HtmlGenerationWebSite_GeneratesExpectedResultsNotReadyForHelix(string action, string antiforgeryPath)
```

Comment:
Note: I'm splitting out the 2 scenarios which currently don't work on helix from this main PR, and we can look into reenabling these in a follow up PR since its just 3 tests